src/reporters/screenshot.reporter.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+
+export class ScreenshotReporter {
+    private static basePath = "./reports/screenshots";
+    private static specCount = 0;
+
+    public static setBasePath(basePath: string): void {
+        this.basePath = basePath;
+    }
+
+    public static beforeSuite(suite: any): void {
+        this.specCount = 0;
+    }
+
+    public static async afterTest(test: { description: string, fullName: string }, context: any,
```

Comment:
Just curiosity, do you now if the context could be used to specify the browser name where the test was executed? Thanks a lot, Jordi! I think that could be very useful for the rest of the projects. :-)